README.md

```diff
@@ -53,13 +53,20 @@ Optional:
  - kickstart_timezone
  - kickstart_log_host
  - dhcp_kickstart_manage_packages
- - kickstart_packages # if dhcp_kickstart_manage_packages is defined then kickstart_packages must contain %packages
+ - kickstart_packages # if dhcp_kickstart_manage_packages is defined, it must contain at least @Core or @Base
```

Comment:
Not sure why change the first part. This form would imply that `dhcp_kickstart_manage_packages` should contain e.g. `@Core`, whereas the code doesn't look like it.